plugins/keep-markup/prism-keep-markup.js

```diff
@@ -100,8 +107,8 @@
 						var range = document.createRange();
 						range.setStart(nodeState.nodeStart, nodeState.nodeStartPos);
 						range.setEnd(nodeState.nodeEnd, nodeState.nodeEndPos);
-						nodeState.node.clone.appendChild(range.extractContents());
-						range.insertNode(nodeState.node.clone);
+						nodeState.node.element.appendChild(range.extractContents());
```

Comment:
@RunDevelopment Here, we could empty the element before appending the range, if we were to store it with its children.
```suggestion
						nodeState.node.element.innerHTML = '';
						nodeState.node.element.appendChild(range.extractContents());
```